homeassistant/components/pi_hole/const.py

```diff
@@ -6,6 +6,7 @@
 
 CONF_STATISTICS_ONLY = "statistics_only"
 
+DEFAULT_API_VERSION = 6
```

Comment:
I have defaulted to 6 as the API version 6 has been out for several months
What I haven't fully appreciated is what happens to someone running a working V5 set up when this update lands.

Reply:
I don't actually see where this variable is used.

Reply:
Indeed, it became defunct once we detected the API version automatically